circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+  pre:
+    - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
+  services:
+    - docker
+
+dependencies:
+  pre:
+    - sudo pip install docker-compose
+    - docker-compose -f docker-compose-integration.yml up -d --force-recreate
+    - mkdir -p ${HOME}/.go_workspace/src/github.com/go-kit/ && ln -sf ${HOME}/kit ${HOME}/.go_workspace/src/github.com/go-kit/
+  override:
+    - cd ${HOME}/.go_workspace/src/github.com/go-kit/kit && go get -t -d -v ./...
```

Comment:
yes, probably with go-kit/kit repo, not with my fork.

before sending the final PR I will remove all this workaround that I need to run builds on circle from my fork.